Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -132,9 +132,12 @@ extension TuistGraph.Target {
         // remove excluding
         try manifest.resources?.excluding.forEach { path in
             guard !path.pathString.contains("/**/") else {
-                throw ResourceFileElementError.globsNotAllowed
+                throw TargetManifestMapperError.invalidResourcesGlob(
+                    targetName: manifest.name,
+                    invalidGlobs: [InvalidGlob(pattern: "/**/", nonExistentPath: AbsolutePath(path.pathString))]
+                )
```

Comment:
I didn't meant to reuse the same error case, but to add a new case to it.

Anyway, as commented by Pedro [here](https://github.com/tuist/tuist/pull/3305#discussion_r694052658) we should update the logic to support globs as well, so the additional error should not be required